Rarime/Code/Modules/ScanPassport/Views/ReadPassportNFCView.swift

```diff
@@ -33,7 +35,12 @@ struct ReadPassportNFCView: View {
                             self.onNext(passport)
                         case .failure(let error):
                             LoggerUtil.passport.error("failed to read passport data: \(error.localizedDescription, privacy: .public)")
-                            self.onBack()
+                            switch error {
+                            case NFCPassportReaderError.ResponseError(let reason, let sw1, let sw2):
```

Comment:
Do you check that the errors has the "Referenced data not found" message? It's the germany error that we discovered.